web-common/src/features/sources/createModel.ts

```diff
@@ -1,11 +1,11 @@
 import { removeLeadingSlash } from "@rilldata/web-common/features/entity-management/entity-mappers";
+import { fetchAllFileNames } from "@rilldata/web-common/features/entity-management/file-selectors";
```

Comment:
**Performance Issue**: Introduction of 'fetchAllFileNames' without filtering for model-specific files could lead to unnecessary processing and performance overhead when creating models from sources. <br> **Fix**: Optimize the 'fetchAllFileNames' function to allow for entity-specific filtering or introduce a new method tailored for model creation scenarios to ensure only relevant files are processed, minimizing performance overhead. <br> **Code Suggestion**: 
 ```
 +import { fetchModelSpecificFileNames } from "@rilldata/web-common/features/entity-management/file-selectors";
 -import { fetchAllFileNames } from "@rilldata/web-common/features/entity-management/file-selectors";
 ```

Reply:
I (real Eric again) also agree with this one. I believe we're okay to use the same name across different resource kinds. So we should only be fetching the model names here.